src/components/images-and-icons/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useMemo } from 'react'
+import { Image, ImageSourcePropType } from 'react-native'
+import Identicon from 'identicon.js'
+import { Theme, useStyles, useTheme } from '../../../theme'
+import { generateSha512 } from '../../../utils/sha512'
+
+export interface GeneratedAvatarSource {
+  /**
+   * The image will be generated from this value using RoboHash.
+   *
+   * @see https://robohash.org/
```

Comment:
Is RoboHash used behind the scenes, or is this a copy-paste error?

https://github.com/stewartlord/identicon.js

Reply:
Nope, remnants from previous commits... :zombie_man: 

Replaced with Identicon.